package.json

```diff
@@ -11,9 +11,10 @@
     "react-native": "https://github.com/expo/react-native/archive/sdk-24.0.0.tar.gz",
     "react-native-collapsible": "~0.10.0",
     "react-native-elements": "~1.0.0-beta4",
-    "react-native-image-gallery": "^2.1.5",
-    "react-native-keyboard-aware-scroll-view": "^0.5.0",
+    "react-native-image-gallery": "~2.1.5",
+    "react-native-keyboard-aware-scroll-view": "~0.5.0",
     "react-native-linear-gradient": "~2.4.0",
+    "react-native-swiper": "~1.5.13",
```

Comment:
Added swiper library for our carousel. Also locked down versions of some more dependencies.